include/bluetooth/gatt_dm.h

```diff
@@ -292,6 +292,11 @@ int bt_gatt_dm_start(struct bt_conn *conn,
  * This function continues service discovery.
  * Call it after the previous data was released by @ref bt_gatt_dm_data_release.
  *
+ * If the service UUID is set when you start service discovery using the
```

Comment:
Please check again my suggestion. This does not really seem right yet. At least the first sentence is a bit weird.

Reply:
I think we can remove this change to the doc. I don't think it adds any new information. If it's not clear that the service discovery will find all matching services, then we should clarify that in the doc of `bt_gatt_dm_start`.